src/main/kotlin/no/nav/syfo/persistering/api/SendRegisterPapirSykmeldingOppgaveRest.kt

```diff
@@ -177,45 +178,59 @@ fun Route.sendPapirSykmeldingManuellOppgave(
+                                        call.respond(HttpStatusCode.InternalServerError)
+                                    }
+                                }
+                            } else {
+                                when (validationResult.status) {
+                                    Status.MANUAL_PROCESSING -> {
+                                        log.info(
+                                            "Ferdigstilling av papirsykmeldinger manuell registering traff regel MANUAL_PROCESSING {}",
+                                            StructuredArguments.keyValue("oppgaveId", oppgaveId)
                                         )
-                                        call.respond(HttpStatusCode.NoContent)
-                                    } else {
-                                        log.error(
-                                            "Ferdigstilling av papirsykmeldinger manuell registering i db feilet {}",
+                                        call.respond(HttpStatusCode.BadRequest, validationResult)
+                                    }
+                                    Status.OK -> {
+                                        log.error("ValidationResult har status OK, men inneholder ruleHits som ikke er hvitelistet",
                                             StructuredArguments.keyValue("oppgaveId", oppgaveId)
                                         )
                                         call.respond(HttpStatusCode.InternalServerError)
                                     }
-                                }
-                                Status.MANUAL_PROCESSING -> {
-                                    log.info(
-                                        "Ferdigstilling av papirsykmeldinger manuell registering traff regel MANUAL_PROCESSING {}",
-                                        StructuredArguments.keyValue("oppgaveId", oppgaveId)
-                                    )
-                                    call.respond(HttpStatusCode.BadRequest, validationResult)
-                                }
-                                else -> {
-                                    log.error("Ukjent status: ${validationResult.status} , papirsykmeldinger manuell registering kan kun ha ein av to typer statuser enten OK eller MANUAL_PROCESSING")
-                                    call.respond(HttpStatusCode.InternalServerError)
+                                    else -> {
+                                        log.error("Ukjent status: ${validationResult.status} , papirsykmeldinger manuell registering kan kun ha ein av to typer statuser enten OK eller MANUAL_PROCESSING")
+                                        call.respond(HttpStatusCode.InternalServerError)
+                                    }
                                 }
```

Comment:
Slik, @tiafiring ?

Reply:
Synes det ser veldig bra ut! 👏  Det hadde kanskje vært bra om det var noen tester på dette, så er det lettere å se om det oppfører seg likt når man refaktorerer. Men for meg ser det riktig ut :)